platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/style/layers/PropertyFactory.java

```diff
@@ -2046,7 +2050,7 @@ public static PropertyValue<Expression> iconRotate(Expression value) {
   }
 
   /**
-   * Size of the additional area around the icon bounding box used for detecting symbol collisions.
+   * Size of additional area round the icon bounding box used for detecting symbol collisions.
```

Comment:
Around seems like the right word, isn't it?

Reply:
It's just copied over.